stl/inc/xmemory

```diff
@@ -2198,6 +2231,31 @@ _NODISCARD_REMOVE_ALG _CONSTEXPR20 _FwdIt remove(_FwdIt _First, const _FwdIt _La
     _UFirst           = _STD _Find_unchecked(_UFirst, _ULast, _Val);
     auto _UNext       = _UFirst;
     if (_UFirst != _ULast) {
+#if _USE_STD_VECTOR_ALGORITHMS
+        if constexpr (_Vector_alg_in_find_is_safe<decltype(_UFirst), _Ty>) {
+#if _HAS_CXX20
+            if (!_STD is_constant_evaluated())
+#endif // _HAS_CXX20

```

Comment:
Can use `_Is_constant_evaluated`.

Reply:
This construct is used everywhere else for `_CONSTEXPR20`

Reply:
Not everywhere:

https://github.com/microsoft/STL/blob/2c9e3e7f10366dbd93a16da73264464c8478cdcb/stl/inc/algorithm#L2121-L2136

Reply:
Ok, maybe it indeed worth doing hereforth. The cost of debug codegen/throughput is tiny, and only for pre-C++20